packages/plugin-react-swc/playground/ts-lib/package.json

```diff
@@ -8,11 +8,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@generouted/react-router": "^1.20.0",
-    "generouted": "1.11.7",
+    "@vitejs/test-dep-non-js": "file:./test-dep/non-js",
```

Comment:
I don't think this as the same behaviour as being in node_modules because node_modules is not in the resolved path